ra-optimisation/linear-rao/src/test/java/com/farao_community/farao/linear_rao/LinearRaoTest.java

```diff
@@ -80,22 +83,43 @@ public void getVersion() {
     }
 
     @Test
-    public void runTest() {
+    public void runTest() throws Exception {
```

Comment:
it is risky to accept any type of Exception here.

Reply:
indeed